packages/zipkin/index.d.ts

```diff
@@ -85,9 +112,48 @@ declare namespace zipkin {
     }
   }
 
+  namespace model {
+    interface Endpoint {
+      constructor(args: { serviceName?: string, ipv4?: InetAddress, port?: number });
+
+      setServiceName(serviceName: string): void;
+      setIpv4(ipv4: InetAddress): void;
```

Comment:
technically this is a string literal which has an ipv4 inside..

Reply:
oh, ipv4 is a string (of a "X.X.X.X" format)?